internal/pkg/plugin/argocd/state.go

```diff
@@ -1,20 +0,0 @@
-package argocd
-
-import "github.com/devstream-io/devstream/pkg/util/helm"
-
-var DefaultDeploymentList = []string{
-	"argocd-applicationset-controller",
-	"argocd-dex-server",
-	"argocd-notifications-controller",
-	"argocd-redis",
-	"argocd-repo-server",
-	"argocd-server",
-}
-
-func GetStaticState() *helm.InstanceState {
-	retState := &helm.InstanceState{}
-	for _, dpName := range DefaultDeploymentList {
-		retState.Workflows.AddDeployment(dpName, true)
-	}
-	return retState
-}
```

Comment:
Why did you put the `state.go` module's code into `argocd.go`, Be consistent with other pacakge may be better

Reply:
because i find this comment in  [514](https://github.com/devstream-io/devstream/issues/514) that says “ internal/pkg/plugin/argocd/argocd.go is empty, not necessary maybe. But since the dtm develop command creates this file, maybe the better thing to do here is to move the logic from the state.go to this file.“ 
and i found there is no state.go in other plugin ,so i remove it's logic into acgocd.go

Reply:
lgtm